prow/cmd/hook/main.go

```diff
@@ -78,6 +79,7 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	fs.StringVar(&o.configPath, "config-path", "", "Path to config.yaml.")
 	fs.StringVar(&o.jobConfigPath, "job-config-path", "", "Path to prow job configs.")
 	fs.StringVar(&o.pluginConfig, "plugin-config", "/etc/plugins/plugins.yaml", "Path to plugin config file.")
+	fs.BoolVar(&o.pluginHelp, "plugin-help", true, "Enable the plugin-help endpoint")
```

Comment:
Could you also add the info why this may not be wanted?

Reply:
Good idea! I updated the help message with an additional note.